src/parser/internal.ts

```diff
@@ -1,13 +1,21 @@
 import * as md from '../markdown';
 import * as notion from '../notion';
 import {URL} from 'url';
-import {LIMITS} from '../notion';
+import {LIMITS, SUPPORTED_CODE_BLOCK_LANGUAGES} from '../notion';
 
 function ensureLength(text: string, copy?: object) {
   const chunks = text.match(/[^]{1,2000}/g) || [];
   return chunks.flatMap((item: string) => notion.richText(item, copy));
 }
 
+function ensureCodeBlockLanguage(lang?: string): string | undefined {
```

Comment:
I found this: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

Is there a more comprehensive/shorter version?

What about adding a simple map?

```
SUPPORTED_CODE_BLOCK_LANGUAGES_MAP = {
  "coffee-script": "coffeescript",
  "coffeescript": "coffeescript"
}
```

Super repetitive but easily editable if you need to update the list in the future.

Reply:
There are some npm packages that expose this data as JSON but I think that a good way to do it would be using a script to compile a map like this at build-time: Notion only supports a small portion of the languages included in linguist's database, so we can save space by compiling our own dictionary instead of using the whole library as a dependency

I'll do it tomorrow 👍🏻

Reply:
Sounds good, feel free to close this if you get that implemented.

Do you have a planned release date for the most recent changes on master?